_posts/2018-08-14-kotlin-tips.md

```diff
@@ -0,0 +1,159 @@
+# Kotlin Tips And Tricks
+
+This post is a series of tips and tickets that [Elliot](https://github.com/elroid) compiled and showed off for his Kotlin Knights meetup in August.
```

Comment:
Would prefer:

This post is a series of tips and observations that [Elliot](http://elroid.com/cv) compiled and presented for our Kotlin Knights meetup in August.